pkg/cmd/run.go

```diff
@@ -67,6 +75,20 @@ func NewRunMicroshiftCommand() *cobra.Command {
 	return cmd
 }
 
+func cleanUpPreviousFileLogs() {
+	for _, p := range cleanUpFileLogPaths {
+		if errLog := p.Remove(); errLog != nil {
+			klog.ErrorS(errLog, "failed to remove log file", "path", p)
+		}
+	}
+}
+
+func writeFileLogError(l util.LogFilePath, err error) {
+	if errLog := l.Write([]byte(err.Error())); errLog != nil {
```

Comment:
Similarly to "log msg review", we might want to do a "error msg review" to make sure that what we write here makes sense, but that's another ticket

Reply:
Yeah good point, didn't want to wrap it and have more repeated words in the output